app/src/main/java/com/antosik/benchproject/app/popular/movies/viewModel/PopularMoviesViewModel.kt

```diff
@@ -37,14 +40,21 @@ class PopularMoviesViewModel @Inject constructor(
         popularMoviesFragmentNavigator.navigateToMovieDetailsFragment(movieId)
     }
 
+    fun onRefreshPopularMovies() {
+        loadPopularMovie()
+    }
+
     private fun loadPopularMovie() {
         viewModelScope.launch {
+            _isPlaceholderVisible.value = false
             _isLoaderVisible.value = true
             val movies = getPopularMoviesUseCase()
             if (movies.isEmpty()) {
                 _isPlaceholderVisible.value = true
+                _isRefreshing.value = false
```

Comment:
Why this line is repeated in `if` and `else`? It makes no sense 😄 just set it once outside the if statement

Reply:
I changed it 😄